tests/providers/elasticsearch/log/test_es_task_handler.py

```diff
@@ -671,14 +671,11 @@ def test_retrieve_config_keys():
     """
     with conf_vars(
         {
-            ("elasticsearch_configs", "use_ssl"): "True",
```

Comment:
I think we need to keep it in the tests for check that we do not use non-existed parameters

Reply:
Which parameter are you worried about?

Reply:
`("elasticsearch_configs", "use_ssl"): "True",` This mostly for check that we remove it from resulting arguments here

Reply:
yes but we no longer need it because we are on `elasticsearch>7`
the only reference to this parameter was in the docs which was wrong